docs/content/release_notes/whats_new.md

```diff
@@ -4,6 +4,28 @@ IBM® block storage CSI driver 1.11.0 introduces the enhancements that are detai
 
 **General availability date:** 15 January 2023
 
+## Enhanced dynamic host definition
+
+The IBM® block storage CSI driver 1.11.0 host definition feature now supports the following:
+
+- **CSI Topology feature**
+
+    For more information, see [Configuring for CSI Topology](../configuration/confiugring_toplogy.md).
+
+- **Dynamically configuring host ports**
+
+     Changes in host port hierarchy are now identified automatically updated. For more inforation, see [Configuring the host definer](../configuration/configuring_hostdefiner.md).
+
+- **I/O groups**
+
+    By default the host definer now creates all definitions across all possible I/O groups. Additionally, an optional label available in order to specify which I/O group should be used on a specific node. For more information, see [Using dynamic host definition](../configuration/configuring_hostdefiner.md).
+
+- **Overriding node host connectivity**
+
+    This version introduces a new label tag, allowing connectivity type definition of a specific node, regardless of connectivity hierarchy. For more information, see [Adding optional labels for dynamic host definition](../using/using_hostdefinition_labels.md).
+    
+In addition, only valid ports are now defined when working with Fibre Channel.
```

Comment:
this is not specifically for FC ports

Reply:
can you add an example that if a host has 4 FC ports and only 2 of them are zoned to the storage, then the host will be created with only those two ports?

Reply:
updated

Reply:
@matancarmeli7  is this OK now? can we close this thread and the PR?